streams/shared/src/main/scala/zio/stream/StreamEffect.scala

```diff
@@ -322,19 +331,22 @@ private[stream] object StreamEffect extends Serializable {
       }
     }
 
-  final def fromInputStream(is: InputStream): StreamEffect[Any, Nothing, Byte] =
-    StreamEffect[Any, Nothing, Byte] {
+  final def fromInputStream(
```

Comment:
I think this needs to go into a JVM-specific file?

Reply:
I am not sure, however scala.js has https://github.com/scala-js/scala-js/blob/master/javalib/src/main/scala/java/io/InputStream.scala

Reply:
Oh well that simplifies everything. We can keep it here then :-)